app/controllers/admin/consultations_controller.rb

```diff
@@ -1,8 +1,9 @@
 class Admin::ConsultationsController < ApplicationController
+  include ImportResponse
```

Comment:
@BALAJI-BALA - why is this being included in the controller? Shouldn't this be something on the model?

Reply:
@mikevic moved to model bro.